app/controllers/logs_controller.rb

```diff
@@ -390,6 +383,20 @@ def export
 
   private
 
+  def parse_start_stop_date(params)
```

Comment:
I might name this `parse_start_stop_dates` as its doing both and the singular bit is not quite clear.